report/finished/Edward_C_evalulation_Effector_Arm.tex

```diff
@@ -0,0 +1,19 @@
+
+\subsection{Robotic arm}
+In the beginning many different arm choices was discussed. One solution was using a simpler forklift design that just moves up and forward in a rigid motion without rotation.
+This solution would suffice. However, we decided to go with a arm design with links connected with motors.
+This is a more complex solution but opens up translation and rotation in 3D-space.
+The design allows the arm to move to any spot in the defined work space.
+to pick up the piece at an angle from factory.
+A limitation is the fact that we decided to keep the of the arm static and parallel to the ground.
+So we can't rotate the end effector. By adding one more motor this can be done. However, we deemed it not necessary for the task at hand.
```

Comment:
"So the end effector cannot rotate". "By adding one extra motor rotation of the en effector can be achived"

Reply:
"it was deemed that it was not necessary", change the "we"